src/plots/mapbox/layout_attributes.js

```diff
@@ -145,10 +165,13 @@ var attrs = module.exports = overrideAll({
             dflt: 'circle',
             role: 'info',
             description: [
-                'Sets the layer type (mapbox.layer.type).',
-                'Support for *raster*, *background* types is coming soon.',
-                'Note that *line* and *fill* are not compatible with Point',
-                'GeoJSON geometries.'
+                'Sets the layer type,',
+                'that is the how the layer data set in `source` will be rendered',
+                'With `sourcetype` set to *geojson*, *circle*, *line*, *fill* and *symbol* are available',
+                'but note that *line* and *fill* are not compatible with Point',
+                'GeoJSON geometries.',
+                'With `sourcetype` set to *vector*, *circle*, *line*, *fill* and *symbol* are available.',
```

Comment:
```suggestion
                'With `sourcetype` set to *vector*, the following values are allowed: *circle*, *line*, *fill* and *symbol*.',
```

Reply:
Now this is:

https://github.com/plotly/plotly.js/blob/29c2bf9ce97403ae844ac0c27c3be3d67da1948d/src/plots/mapbox/layout_attributes.js#L170-L178